README.md

```diff
@@ -1,602 +1,370 @@
+expressed in plain English, or in other words, _specs._
 
-### Generators
+Use **[`rspec-rails` 1.x][]** for Rails 2.x.
 
-Once installed, RSpec will generate spec files instead of Test::Unit test files
-when commands like `rails generate model` and `rails generate controller` are
-used.
+[Build Status]: https://secure.travis-ci.org/rspec/rspec-rails.svg?branch=master
+[travis-ci]: http://travis-ci.org/rspec/rspec-rails
+[Code Climate]: https://img.shields.io/codeclimate/github/rspec/rspec-rails.svg
+[code-climate]: https://codeclimate.com/github/rspec/rspec-rails
+[RSpec]: http://rspec.info/
+[Ruby on Rails]: https://rubyonrails.org/
+[`rspec-rails` 1.x]: http://github.com/dchelimsky/rspec-rails
 
-You may also invoke RSpec generators independently. For instance,
-running `rails generate rspec:model` will generate a model spec. For more
-information, see [list of all
-generators](https://www.relishapp.com/rspec/rspec-rails/docs/generators).
+## Installation
 
-## Contributing
+1. Add `rspec-rails` to **both** the `:development` and `:test` groups
+   of your app’s `Gemfile`:
 
-Once you've set up the environment, you'll need to cd into the working
-directory of whichever repo you want to work in. From there you can run the
-specs and cucumber features, and make patches.
+   ```ruby
+   # Run against the latest stable release
+   group :development, :test do
+     gem 'rspec-rails', '~> 3.8'
+   end
 
-NOTE: You do not need to use rspec-dev to work on a specific RSpec repo. You
-can treat each RSpec repo as an independent project.
-Please see the following files:
+   # Or, run against the master branch
+   # (requires master-branch versions of all related RSpec libraries)
+   group :development, :test do
+     %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
+       gem lib, :git => "https://github.com/rspec/#{lib}.git", :branch => 'master'
+     end
+   end
+   ```
 
-For `rspec-rails`-specific development information, see
+   (Adding it to the `:development` group is not strictly necessary,
+   but without it, generators and rake tasks must be preceded by `RAILS_ENV=test`.)
 
-- [Build details](BUILD_DETAIL.md)
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Detailed contributing guide](CONTRIBUTING.md)
-- [Development setup guide](DEVELOPMENT.md)
+2. Then, in your project directory:
 
+   ```sh
+   # Download and install
+   $ bundle install
 
-## Model Specs
+   # Generate boilerplate configuration files
+   # (check the comments in each generated file for more information)
+   $ rails generate rspec:install
+         create  .rspec
+         create  spec
+         create  spec/spec_helper.rb
+         create  spec/rails_helper.rb
 
-Use model specs to describe behavior of models (usually ActiveRecord-based) in
-the application.
+   # Rails 4+ only: generate a binstub
+   # (so you can run `bin/rspec` instead of `bundle exec rspec`)
+   $ bundle binstubs rspec-core
```

Comment:
@JonRowe I honestly *never* use binstubs, I don't know if we want this to be the default advice?

Reply:
I don't use them, but Rails does, so I feel like maybe this advice should remain, happy to take you lead on this though

Reply:
I sort of want this to be in here, but de-emphasize it. Maybe pull it out in to a separate code box as an optional step?

Reply:
Ok by me

Reply:
@rlue could you pull this out in to a separate code block and say that it's optional if you prefer binstubs?

Reply:
I actually went so far as to move it to the Usage section. My reasoning here is that before you've seen examples of `bundle exec rspec` usage, it may be difficult to grasp why you might even want a binstub in the first place.

As an ancillary benefit, it distills the Installation section down to the barest minimum.